zebra/zebra_vty.c

```diff
@@ -395,15 +394,21 @@ DEFPY(ip_route_address_interface,
       "IP destination prefix\n"
       "IP destination prefix mask\n"
       "IP gateway address\n"
-      "IP gateway interface name\n"
+      "IP gateway interface name. Specify 'Null0' (case-insensitive) for a \
+      null route.\n"
       "Set tag for this route\n"
       "Tag value\n"
       "Distance value for this route\n"
       VRF_CMD_HELP_STR
       MPLS_LABEL_HELPSTR)
 {
+	const char *flag = NULL;
+	if (!strncasecmp(ifname, "Null0", 5)) {
```

Comment:
`ifname` might be null (example: ip route 50.0.0.0/8 10.0.1.2)

Reply:
ah, yep. thanks